From 604cca8d9e56f396b48cc1059133e17d1daf5c0d Mon Sep 17 00:00:00 2001
From: Nicolai Buchwitz <n.buchwitz@kunbus.com>
Date: Sun, 2 Feb 2025 15:35:41 +0100
Subject: [PATCH] configs: Enable Renesas XHCI driver

Renesas UPD720201/-202 is a popular alternative to the VL805 USB3 phy
used in many CM4 based products.

Commit 25f51b76f90f ("xhci-pci: Make xhci-pci-renesas a proper modular
driver") reworked the Renesas XHCI driver, resulting in
CONFIG_USB_XHCI_PCI_RENESAS no longer being implicitly enabled.

Explicitly add it to the defconfig to restore USB3 functionality.

Signed-off-by: Nicolai Buchwitz <n.buchwitz@kunbus.com>
---
 arch/arm/configs/bcm2711_defconfig   | 1 +
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcm2712_defconfig | 1 +
 3 files changed, 3 insertions(+)

--- a/arch/arm/configs/bcm2711_defconfig
+++ b/arch/arm/configs/bcm2711_defconfig
@@ -1177,6 +1177,7 @@ CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_MON=m
 CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_PCI_RENESAS=m
 CONFIG_USB_XHCI_PLATFORM=y
 CONFIG_USB_DWCOTG=y
 CONFIG_USB_PRINTER=m
--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -1232,6 +1232,7 @@ CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_MON=m
 CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_PCI_RENESAS=m
 CONFIG_USB_DWCOTG=y
 CONFIG_USB_PRINTER=m
 CONFIG_USB_TMC=m
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -1234,6 +1234,7 @@ CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
 CONFIG_USB_MON=m
 CONFIG_USB_XHCI_HCD=y
+CONFIG_USB_XHCI_PCI_RENESAS=m
 CONFIG_USB_DWCOTG=y
 CONFIG_USB_PRINTER=m
 CONFIG_USB_TMC=m
